src/mongo/db/views/durable_view_catalog.h

```diff
@@ -65,6 +65,7 @@ class DurableViewCatalog {
                         const BSONObj& view) = 0;
     virtual void remove(OperationContext* opCtx, const NamespaceString& name) = 0;
     virtual const std::string& getName() const = 0;
+    virtual ~DurableViewCatalog() {}
```

Comment:
Thanks for the pull request!  One observation: I believe "= default;" is now preferred in our C++11-style codebase.